app/admin/councillor_contributions.rb

```diff
@@ -10,10 +10,9 @@
   end
 
   show title: :created_at do
-    attributes_table do
-      row :contributor
-      row :id
-      row :authority
+
+    panel "Councillor Contribution Details" do
+      render partial: "details", locals: {contributor: resource.contributor, authority: resource.authority}
```

Comment:
If the contributor doesn't submit their contact details this page will now fail (the exception is `undefined method 'name' for nil:NilClass`.